nhsuk-design-system/config/router.js

```diff
@@ -287,18 +289,6 @@ router.get('/examples/google-allergies', (req, res) => {
   res.render('examples/google-allergies/index', {});
 });
 
-router.get('/examples/test-template-general', (req, res) => {
```

Comment:
Are these meant to have been deleted?

Reply:
no, eek. how do i fix?

Reply:
Probably easiest thing is to see what was deleted here (red background content):
https://github.com/nhsuk/frontend/pull/125/files
and copy it back in to your file, then commit that.